back/config/environments/test.rb

```diff
@@ -74,4 +74,12 @@
   # config.rails_semantic_logger.started    = true
   # config.rails_semantic_logger.processing = true
   # config.rails_semantic_logger.rendered   = true
+
+  # Uncomment to catch N+1 queries when running tests
+  # config.after_initialize do
+  #   Bullet.enable = true
+  #   Bullet.rails_logger = true
+  #   Bullet.bullet_logger = true
+  #   Bullet.raise = true # for testing
+  # end
```

Comment:
Added this for convenience so I can quickly turn on to catch n+1 queries when running tests